mantle/platform/qemu.go

```diff
@@ -312,7 +312,7 @@ func (inst *QemuInstance) SwitchBootOrder() (err2 error) {
 	}
 	// Get boot device (for iso-installs) and block device
 	for _, dev := range blkdevs.Return {
-		devpath := filepath.Clean(strings.Trim(dev.DevicePath, "virtio-backend"))
+		devpath := filepath.Clean(dev.DevicePath)
```

Comment:
The original code wasn't doing what it thought it was, but it still would have removed the string.  Do we know that that's not required?

Reply:
AFAICT playing with the QEMU monitor, it isn't. But I suspect there might be multi-arch factors at play here, so I just implemented what I think the code actually intended.